src/front/shared/components/FAQ/styles.scss

```
@@ -107,6 +95,14 @@ $chrestStripHeight: 12px;
   }
 }
 
+.descriptionFee {
+  margin: 0;
+
+  b {
+    font-family: 'Lucida Console', Monaco, monospace;
```

```suggestion
    opacity: 0.7;
    font-family: 'Roboto Mono', monospace;
```

As used in https://github.com/swaponline/MultiCurrencyWallet/blob/master/src/front/shared/pages/Exchange/Exchange.scss#L284